### ISSUE TITLE
Syntactic sugar/documentation for parameter sharing between different bricks

### ISSUE THREAD

**Issue by dwf** [mila-udem/blocks#731]
See [this mailing list thread](https://groups.google.com/forum/#!topic/blocks-users/9ZcqcKn9yRo).

**Comment by vikkamath** [comment]
Is there a recommended course of action? Should it be an added feature to make parameter sharing easier or should a 'recipe' be added to the docs?

**Comment by dwf** [comment]
I don't think we have a clear course of action yet. The docs are probably a bare minimum, and if someone has some proposal for making it simpler then they should probably bring it up here or on the mailing list.
